New_PlatformGame/Assets/1. Script/GameManager.cs

```diff
@@ -72,8 +76,7 @@ void SpawnScaffolding()
     
     void MissileSummoning() //미사일 생성
     {
-        CameraPositionTopRight = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width,Screen.height, 0));
-        MissileRespawn = new Vector3(Random.Range(CameraPositionBottomLeft.x, CameraPositionTopRight.x), CameraPositionTopRight.y,0);
-        Instantiate(Missile, MissileRespawn, Quaternion.identity);
+        CameraPositionTopRight = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width, Screen.height, 0));
+        Instantiate(Missile, new Vector3(MissileRespawn.x, CameraPositionTopRight.y, 0), Quaternion.identity);
     }
```

Comment:
```suggestion
    void MissileSummoning() //미사일 생성
    {
        CameraPositionTopRight = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width,Screen.height, 0));
        MissileRespawn = new Vector3(Random.Range(CameraPositionBottomLeft.x, CameraPositionTopRight.x), CameraPositionTopRight.y,0);
        Instantiate(Missile, MissileRespawn, Quaternion.identity);
        CameraPositionTopRight = Camera.main.ScreenToWorldPoint(new Vector3(Screen.width, Screen.height, 0));
        Instantiate(Missile, new Vector3(MissileRespawn.x, CameraPositionTopRight.y, 0), Quaternion.identity);
    }
```
사용되지 않는 메소드는 제거 바랍니다.